### PR TITLE
docs: update to the streaming tutorial notebook in the lcel documentation

### PR DESCRIPTION
**Pull request by sepiatone**
**Description:** Update to the streaming tutorial notebook in the LCEL documentation
**Issue:** Fixed an import and (minor) changes in documentation language
**Dependencies:** None

### CODE REVIEW

File: docs/docs/expression_language/streaming.ipynb
Comment (baskaryan) [inline docs/docs/expression_language/streaming.ipynb:1]: could we add a comment above langchain_anthropic import saying you need to install it with something like `pip install -qU langchain-anthropic`

Comment (sepiatone) [reply docs/docs/expression_language/streaming.ipynb:1]: @baskaryan 

Made the suggested changes

**Comment by sepiatone** [comment]
test

**Comment by sepiatone** [comment]
@baskaryan Update with the requested changes, please review

**Review by baskaryan — APPROVED** [review]

**Pull request merged**